.github/workflows/pnp-input-api-build.yml

```diff
@@ -316,12 +316,21 @@ jobs:
               "apikey": "${{ env.NUGET_API_KEY }}"
             }]
 
+      - name: Get Identity token
+        id: identitytoken
+        uses: extenda/actions/identity-token@v0
+        with:
+          service-account-key: ${{ secrets.GCLOUD_AUTH_STAGING }}
+          service-account: braveheart-quotes-webclient@quotes-staging-ccdf.iam.gserviceaccount.com
+          audiences: bhq-braveheart-quotes
+
       - name: Run Acceptance Tests
         run: |
           dotnet test ${{ secrets.path-to-acceptance-tests }}
         env:
           INPUT_API_URL: ${{ secrets.staging-service-base-url }}
           BEARER_TOKEN: ${{ steps.testtoken.outputs.iam-token }}
+          IDENTITY_TOKEN: ${{ steps.identitytoken.outputs.identity-token }}
```

Comment:
Maybe it makes sense to run the same set of tests with WID token? Or how did you plan to use this env variable in the acceptance tests?

Reply:
Ok